INSTALL

```diff
@@ -1,5 +1,3 @@
 Building Zcash
 
-See the Zcash documentation wiki (https://zcash.readthedocs.io/en/latest/rtd_pages/zcashd.html) for instructions on building zcashd,
-the intended-for-services, no-graphical-interface, reference
-implementation of Zcash.
+See [the Zcashd book](doc/book/src/) for installation, build, and operation instructions.
```

Comment:
We should link to the rendered version in this case (as this file is aimed at users, not contributors):
```suggestion
See [the `zcashd` book](https://zcash.github.io/zcash/) for installation, build, and operation instructions.
```